packages/cowsay-beefymiracle/SPECS/cowsay-beefymiracle.spec

```diff
@@ -0,0 +1,80 @@
+Name:           cowsay-beefymiracle
+Version:        1.0
+Release:        13%{?dist}
+Summary:        Cowsay file for the Beefy Miracle
+
+License:        CC-BY-SA
+URL:            http://rrix.fedorapeople.org/beefsay
+Source0:        http://rrix.fedorapeople.org/beefsay/beefymiracle.cow
+
+BuildRequires:  cowsay
+Requires:       cowsay
+
+BuildArch:      noarch
+
+%description
+Provides a cowsay file for His Holiness the Beefy Miracle. It can be invoked
+using cowsay -f beefymiracle, or aliased appropriately.
+
+%prep
+# Nothing to do
+
+%build
+# Nothing to do
+
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/%{_datadir}/cowsay/
+install %SOURCE0 %{buildroot}/%{_datadir}/cowsay/beefymiracle.cow
+sed -i -e 's,/bin/env perl,/usr/sgug/bin/env perl,g' %{buildroot}/%{_datadir}/cowsay/beefymiracle.cow
+
+
+%check
+export COWPATH=%{buildroot}%{_datadir}/cowsay
+echo Installation successful | cowsay -f beefymiracle
+
+
+%files
+%doc
+%{_datadir}/cowsay/beefymiracle.cow
+
+%changelog
```

Comment:
Add a changelog entries when you get a chance 😎

Reply:
👍